### DISCUSSION TITLE
Drag-and-Drop: Can not drag bullet/task to another list

### DISCUSSION THREAD

**Discussion by exceptionraiser** [vslinko/obsidian-outliner#465] [Ideas]
Great addition to Outliner plugin!

One issue I see is that you cannot drag a bullet from one list to another. Moving tasks/bullets between lists is something I personally do several times a day as part of my workflow. I would think other probably do the same depending on how you manage tasks.

![CleanShot 2023-05-23 at 12 31 58](https://github.com/vslinko/obsidian-outliner/assets/112484/984a9025-5d89-4a92-b3fd-17b4e4838767)

**Comment by kimaldis** [comment]
+1 from me.

**Comment by sir-stefano** [comment]
+1

**Comment by deadrubbish** [comment]
+1 

and, even better to be able to drag between panes - then could have clone of current note in side bar and drag between lists/ nodes even with long lists/multiple nodes on same doc.

Excellent plugin though - thank you
